tests/testcases/Group8-Configuration/8-01-Authentication-edit-authentication.md

```diff
@@ -14,11 +14,15 @@ User guide
 
 # Test Steps:
 
-TBD
+1. Login UI as admin user.  
+2. In configuration page,change authentication mode from DB to LDAP or from LDAP to DB.  
+3. Save the configuration.  
+4. Change the configuration again.  
 
 # Expected Outcome:
 
-TBD
+* In step2, user can change authentication mode.  
+* In step4, user cannot change authentication mode.

```

Comment:
The auth mode can not be changed anymore only when new users have been added into the system. So if there is only admin user, admin can change auth mode many times.